torch/csrc/autograd/python_variable.cpp

```diff
@@ -1010,6 +1094,70 @@ static c10::SymDimVector tuple_to_symintlist(PyObject* obj) {
   return res;
 }
 
+static constexpr c10::DispatchKeySet after_Python_keyset =
+    c10::DispatchKeySet(c10::DispatchKeySet::FULL) ^
+    (c10::DispatchKeySet(
+         c10::DispatchKeySet::FULL_AFTER,
+         c10::DispatchKey::Python) |
+     c10::DispatchKeySet(c10::DispatchKey::Python));
+
+// This is much simpler than the __torch_function__ paths precisely
+// because it doesn't have to worry about anything to do with dispatch
+// or __torch_function__.
+void callDTensorCustomOpHandler(
```

Comment:
hmm, given that we have to respect monkey patching of _custom_op_handlers, I think that having the explicit hardcoded custom op handler path 1) is confusing 2) doesn't respect monkey patching that removes the hardcoded op handlers. I'll remove it.